railties/lib/rails/generators/rails/app/templates/dockerignore.tt

```diff
@@ -47,6 +47,12 @@
 /.github
 <% end -%>
 
+<% unless options.skip_kamal? -%>
+# Ignore Kamal files.
+/config/deploy.yml
+/.kamal
+<% end -%>
+
```

Comment:
Moving `<% end -%>` to the next line will will prevent a double empty line when Kamal is skipped:

```suggestion

<% end -%>
```

Reply:
Ah - fixed whitespace.